lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1204,41 +1204,96 @@ def cmd_route_tabs(str, words)
   end
 
   def cmd_save_help
-    print_line "Usage: save"
+    print_line "Usage:"
     print_line
-    print_line "Save the active datastore contents to disk for automatic use across restarts of the console"
+    print_line "save:                       save the active module options to disk for automatic use across restarts of the console"
+    print_line "save -d / --delete:         delete the active module options from disk without refreshing current options"
+    print_line "save -r / --reload:         reload default options for the active module"
+    print_line "save -l / --load:           load the saved options for the active module"
+    print_line "save -D / --delete-all:     delete ALL datastore options from the config file"
```

Comment:
Most users wouldn't know what datastore objects are IMO, so something like this may translate better and let the user know this will wipe more than just module info
```suggestion
    print_line "save -D / --delete-all:     deletes ALL saved options from the config file, including DB information"
```

Reply:
Yup, makes sense.
I think I might have to change this over to not delete the likes of
```
[framework/database]
default_db=local-https-data-service

[framework/database/local-https-data-service]
url=https://localhost:5443
```
as that contains quite valuable information and an API key. 🤔

Reply:
Yeah I think that's a save bet. Change the description and rename `--delete-all` to `--delete-all-modules` to make it's behaviour explicit and you're good to go 👍

Reply:
Are you able to discern module information from other framework information reliably?

Reply:
Schweet

Reply:
Framework options are no longer being deleted since the latest commit 👍